src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -27,6 +29,8 @@
     private final String name;
     private final String phone;
     private final String email;
+    private final String telegram;
```

Comment:
Maybe this could be named telegramHandle, a few different stuff being named telegram already

Reply:
I defined the class as `Telegram` and follow this naming convention as well when naming the `telegram` field for `Person`.

What other different stuff are named `telegram`?

Reply:
I think telegram is clear enough. `Telegram` represents the class, while `telegram` is clear enough to understand as handle.